charts/newrelic-infrastructure/templates/kubelet/daemonset.yaml

```diff
@@ -251,8 +251,9 @@ spec:
       tolerations:
         {{- . | nindent 8 }}
       {{- end }}
-      {{- with .Values.kubelet.nodeSelector | default (fromYaml (include "newrelic.common.nodeSelector" .)) }}
       nodeSelector:
+        kubernetes.io/os: linux 
```

Comment:
do we know how recent was this label introduced to k8s? would using it affect clusters using older versions?